cmd/mayactl/app/command/volume_info.go

```diff
@@ -0,0 +1,102 @@
+package command
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	client "github.com/openebs/maya/pkg/client/jiva"
+	"github.com/openebs/maya/pkg/util"
+	"github.com/openebs/maya/types/v1"
+	"github.com/spf13/cobra"
+)
+
+var (
+	volumeInfoCommandHelpText = `
+	    Usage: mayactl volume info --volname <vol>
+
+            This command queries the info of the volume.
+        `
+)
+
+// CmdVolumeAccessOptions is used to store the value of flags used in the cli
```

Comment:
Please review the comments (like this in all the changed files) to make sure they are reflecting the name changes.